src/ja/server/database/sql/database.py

```diff
@@ -177,6 +182,10 @@ def find_job_by_id(self, job_id: str) -> Optional[DatabaseJobEntry]:
                 jobs_entry.statistics.paused_time \
                     = (datetime.now() - jobs_entry.statistics.time_started).seconds - jobs_entry.statistics.running_time
             session.commit()
+            logger.info("job entry with job id: %s found." % job_id)
+            logger.debug("%s" % str(jobs_entry.job))
```

Comment:
just `logger.debug(str(jobs_entry.job))` ?

Reply:
but then if the level is set to INFO you wont get any logging.